src/Services/Webhooks/Webhooks.API/Controllers/WebhooksController.cs

```diff
@@ -71,14 +71,14 @@ public async Task<IActionResult> SubscribeWebhook(WebhookSubscriptionRequest req
         }
         else
         {
-            return StatusCode(418, "Grant url can't be validated");
+            return StatusCode(StatusCodes.Status418ImATeapot, "Grant url can't be validated");
```

Comment:
I assume the status code used is for fun, but the error message reads wrong.

```suggestion
            return StatusCode(StatusCodes.Status418ImATeapot, "Grant URL invalid");
```